### PR TITLE
Explain "shared-storage-select-url" and "private-aggregation" permissions policy

### PR DESCRIPTION
**Pull request by xyaoinum**
We'd like more fine-grained control of the output gates

### CODE REVIEW

**Review by Milujem — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -246,7 +246,14 @@ After a document dies, the corresponding worklet will be kept alive for maximum
 
 ## Permissions Policy
 
-Shared storage methods can be disallowed by the "shared-storage" [policy-controlled feature](https://w3c.github.io/webappsec-permissions-policy/#policy-controlled-feature). Its default allowlist is * (i.e. every origin). 
+Shared storage methods can be disallowed by the "shared-storage" [policy-controlled feature](https://w3c.github.io/webappsec-permissions-policy/#policy-controlled-feature). Its default allowlist is * (i.e. every origin).
+
+The sharedStorage.selectURL() method can be disallowed by the "shared-storage-select-url" [policy-controlled feature](https://w3c.github.io/webappsec-permissions-policy/#policy-controlled-feature). Its default allowlist is * (i.e. every origin).
+
+### Permissions Policy inside the shared storage worklet
+The permissions policy inside the shared storage worklet will be identical to the permissions policy on the associated document.
```
Comment (alexmturner) [inline README.md:254]: maybe: s/will be identical to/will inherit/

Comment (xyaoinum) [reply README.md]: Done.

File: README.md [hunk 2]
```diff
@@ -246,7 +246,14 @@ After a document dies, the corresponding worklet will be kept alive for maximum
 
 ## Permissions Policy
 
-Shared storage methods can be disallowed by the "shared-storage" [policy-controlled feature](https://w3c.github.io/webappsec-permissions-policy/#policy-controlled-feature). Its default allowlist is * (i.e. every origin). 
+Shared storage methods can be disallowed by the "shared-storage" [policy-controlled feature](https://w3c.github.io/webappsec-permissions-policy/#policy-controlled-feature). Its default allowlist is * (i.e. every origin).
+
+The sharedStorage.selectURL() method can be disallowed by the "shared-storage-select-url" [policy-controlled feature](https://w3c.github.io/webappsec-permissions-policy/#policy-controlled-feature). Its default allowlist is * (i.e. every origin).
+
+### Permissions Policy inside the shared storage worklet
+The permissions policy inside the shared storage worklet will inherit the permissions policy on the associated document.
```
Comment (jkarlin) [inline README.md:254]: s/on the/of the/

Comment (xyaoinum) [reply README.md]: Done.

**Pull request merged**